docs/blocks-storage/querier.md

```diff
@@ -878,4 +878,9 @@ blocks_storage:
     # will be stored. 0 or less means disabled.
     # CLI flag: -blocks-storage.tsdb.max-exemplars
     [max_exemplars: <int> | default = 0]
+
+    # [EXPERIMENTAL] Configures the maximum capacity for out-of-order chunks (in
```

Comment:
I suggest following reword: 

```
Configures the maximum number of samples that can be out-of-order.  See [some link] on how out of order works.  
```

I think we somehow need to link to a document that talks about https://docs.google.com/document/d/1Kppm7qL9C-BJB1j6yb6-9ObG3AbdZnFUBYPNNWwDBYM/edit and https://github.com/prometheus/prometheus/pull/11075 because of the experimental feature. 

I am thinking we should have a doc in `cortexmetrics.io` about OOO support. WDYT? I am more than happy to work on the documentation or OOO support.

Reply:
Also, `if set to <=0, default value 32 is assumed.` is this what Prometheus do? My preference is not to do this. I would prefer my application fail to start if I configure some nonsense value because its more explicit.

Reply:
> Configures the maximum number of samples that can be out-of-order.

This looks good. I think I will also mention this per chunk.
Yeah I love the idea of having a doc about this feature for sure.
And I added a validation to make sure it is > 0.